src/as-validator-issue-tag.h

```diff
@@ -921,6 +921,26 @@ AsValidatorIssueTag as_validator_issue_tag_list[] =  {
 	  N_("This color is not a valid HTML color code."),
 	},
 
+	{ "custom-invalid-tag",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("The custom tags can only contains value tags"),
```

Comment:
Again, any literal should be in backticks so translators don't translate it by accident.